salt/_modules/metalk8s_kubernetes.py

```diff
@@ -299,6 +299,22 @@ def method(manifest=None, name=None, kind=None, apiVersion=None,
 update_object = _object_manipulation_function('update')
 
 
+# Check if a specific object exists
+def object_exists(kind, apiVersion, name, **kwargs):
+    """
+    Simple helper to check if an object exists or not
+
+    CLI Examples:
+
+    .. code-block:: bash
+
+        salt-call metalk8s_kubernetes.object_exists kind="Node" apiVersion="v1" name="MyNode"
+    """
+    return bool(
```

Comment:
Any falsy result outside of `None` would be a bug, so let's make this method somewhat more obvious:

```python
return get_object(...) is not None
```